demos/core-us-investment-style/demo.js

```diff
@@ -65,8 +65,86 @@ async function displayXRayUS (postmanJSON) {
 
     await connector.load();
 
-    // eslint-disable-next-line no-console
-    console.log(connector.dataTables);
+    Highcharts.chart('container', {
+        chart: {
+            type: 'heatmap'
+        },
+        title: {
+            text: 'XRay US Equity Style - Example Portfolio',
+            align: 'left'
+        },
+        subtitle: {
+            text: 'Stock Style',
+            align: 'left'
+        },
+        colorAxis: {
+            dataClasses: [
+                { from: 49, color: '#014ce5', name: '50+' },
+                { from: 24, to: 49, color: '#487cea', name: '25-49' },
+                { from: 9, to: 24, color: '#acc2f3', name: '10-24' },
+                { from: 0, to: 9, color: '#fafafa', name: '0-9' }
+            ]
+        },
+        xAxis: {
+            categories: ['Value', 'Blend', 'Growth'],
+            lineWidth: 0,
+            title: {
+                text: 'Style'
+            },
+            gridLineWidth: 0,
+            opposite: true,
+            labels: {
+                style: {
+                    fontSize: '1rem',
+                    color: '#6e7481'
+                }
+            }
+        },
+        yAxis: {
+            categories: ['Small', 'Medium', 'Large'],
+            gridLineWidth: 0,
+            title: {
+                text: 'Size'
+            },
+            labels: {
+                rotation: -90,
+                style: {
+                    fontSize: '1rem',
+                    color: '#6e7481'
+                }
+            }
+        },
+        legend: {
+            layout: 'vertical',
+            verticalAlign: 'top',
+            align: 'right',
+            y: 75,
+            symbolRadius: 0,
+            itemMarginTop: 9,
+            itemMarginBottom: 9
+        },
+        tooltip: {
+            pointFormat: `
+                <b> {series.yAxis.categories.(point.y)}
+                    {series.xAxis.categories.(point.x)
```

Comment:
Missing right-hand side closing bracket.
```suggestion
                    {series.xAxis.categories.(point.x)}
```